### PR TITLE
[BUGFIX] replace non-breaking-spaces and tabs like normal spaces

### PR DESCRIPTION
**Pull request by rvock**
Treat all whitespace-characters the same.

### CODE REVIEW

**Comment by rvock** [comment]
Fixes #218

File: Classes/Utility.php [hunk 1]
```diff
@@ -73,7 +73,8 @@ public function convertToSafeString($processedTitle, $spaceCharacter = '-', $str
 			$processedTitle = $this->csConvertor->conv_case('utf-8', $processedTitle, 'toLower');
 		}
 		$processedTitle = strip_tags($processedTitle);
-		$processedTitle = preg_replace('/[ \-+_]+/', $spaceCharacter, $processedTitle);
+		// the space in the next line is a non-breaking space
+		$processedTitle = preg_replace('/[ \s\-+_]+/u', $spaceCharacter, $processedTitle);
```
Comment (dmitryd) [inline Classes/Utility.php]: I afraid this can be lost if somebody copy/pastes, etc. Can we do using `\x` sequences?

**Comment by rvock** [comment]
I changed the invisble non-breaking space to \xa0.

File: Classes/Utility.php [hunk 2]
```diff
@@ -73,7 +73,8 @@ public function convertToSafeString($processedTitle, $spaceCharacter = '-', $str
 			$processedTitle = $this->csConvertor->conv_case('utf-8', $processedTitle, 'toLower');
 		}
 		$processedTitle = strip_tags($processedTitle);
-		$processedTitle = preg_replace('/[ \-+_]+/', $spaceCharacter, $processedTitle);
+		// \x20 is a non-breaking space
```
Comment (dmitryd) [inline Classes/Utility.php]: Are you sure? :)

**Comment by dmitryd** [comment]
http://stackoverflow.com/questions/12837682/non-breaking-utf-8-0xc2a0-space-and-preg-replace-strange-behaviour ?

**Pull request closed without merging**